File: result/translate/starchat/java2python/python/FIND_LAST_DIGIT_FACTORIAL_DIVIDES_FACTORIAL_B.py
def f_gold(A, B):
        if A == B:
            return 1
        if (B - A) >= 5:
            return 0
        variable = 1
        for i in range(A + 1, B + 1):
            variable = (variable * (i % 10)) % 10
        return variable % 10

File: result/repair/transcoder/python2java/java_repair/FIND_LAST_DIGIT_FACTORIAL_DIVIDES_FACTORIAL_B.java
  int f_gold ( int A , int B ) {
  int variable = 1 ;
  if ( A == B ) {
    return 1 ;
  }
  else if ( ( B - A ) >= 5 ) {
    return 0 ;
  }
  else {
    for ( int i = A + 1 ;
    i <= B ;
    i ++ ) {
      variable = ( variable * ( i % 10 ) ) % 10 ;
    }
    return variable % 10 ;
  }
}

